qiskit_ibm_runtime/utils/backend_converter.py

```diff
@@ -349,11 +320,3 @@ def qubit_props_list_from_props(
             )
         )
     return qubit_props
-
-
-def is_fractional_gate(gate: Gate) -> bool:
-    """Test if gate is fractional gate familiy."""
-    # In IBM architecture these gates are virtual-Z and delay,
-    # which don't change control parameter with its gate parameter.
-    exclude_list = (RZGate, PhaseGate, U1Gate, Delay)
-    return len(gate.params) > 0 and not isinstance(gate, exclude_list)
```

Comment:
Since the code here is IBM-specific, I went the simpler route of just hardcoding which gates count as fractional. We could discuss that if necessary. At the point where the filtering is done now, the gates are just strings, though they could be mapped to Qiskit gates.